src/phoenix/server/main.py

```diff
@@ -116,6 +146,8 @@ def _get_pid_file() -> Path:
     parser.add_argument("--umap_params", type=str, required=False, default=DEFAULT_UMAP_PARAMS_STR)
     parser.add_argument("--debug", action="store_false")
     parser.add_argument("--enable-prometheus", type=bool, default=False)
+    parser.add_argument("--server-trace-collector-http-endpoint", type=str, required=False)
+    parser.add_argument("--server-trace-collector-grpc-endpoint", type=str, required=False)
```

Comment:
Rather than command line arguments, can we make these configurable via environment variables? That will be easier for someone using the Docker container to configure, otherwise, they need to overwrite the `CMD` at runtime.

Reply:
Might make sense to support a comma-separate list of URLs in case someone wants to log to multiple collectors.